scripts/stats

```diff
@@ -1,8 +1,6 @@
-#!/bin/bash
-
-cd /var/local
-
-source venvs/tildetown/bin/activate
-output=$(python tildetown/tildetown-py/stats.py)
-deactivate
-echo $output
+#!/bin/sh
+# run stats.py in the appropriate virtualenv.
+#
+# invoke periodically from crontab and direct the output to
+# /var/www/tilde.town/tilde.json
+exec /usr/local/virtualenvs/tildetown/bin/python /usr/local/tildetown-scripts/tildetown/stats.py
```

Comment:
Yes: the main thing "activate" does is change $PATH so when you type "python" it uses the one in the venv's bin/. No matter how you launch it, that python always uses the modules in its venv. So this works fine.

Where it would break: if whatever you ran from the venv's `bin/` wanted to execute as a subprocess another executable that lives in the venv's `bin/`, _and also_ it didn't specify the path to it explicitly. Then, it's easier to use `activate`.